azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/VirtualNetworkGatewayConnection.java

```diff
@@ -144,7 +145,7 @@
 
             WithSecondVirtualNetworkGateway withVNetToVNet();
 
-            WithExpressRoute withExpressRoute();
+            WithExpressRouteCircuit withExpressRoute();
```

Comment:
missing javadocs for all these withers and interfaces. (I wonder why this passed checkstyle)